setup/paths.sh

```diff
@@ -37,38 +37,27 @@ else
     source $BUILD_HOME/cmsgemos/setup/paths.sh
 fi
 
+# Export project
+export GEM_PLOTTING_PROJECT=$BUILD_HOME/gem-plotting-tools
```

Comment:
I think this variable name is fine, it matches the convention declared in:

- `cmsgemos`
- `vfatqc-python-scripts`

And allows for easy reference in future development.